src/commands/prompt/commands.ts

```diff
@@ -34,30 +27,27 @@ export function processCommand(input: string, context: CommandContext): boolean
     output.outputInfo('Available commands:');
     output.outputInfo('- /exit: Exit the CLI');
     output.outputInfo('- /info: Show current provider, model, and system prompt');
-    output.outputInfo('- /verbose [on|off]: Enable or disable verbose output');
-    output.outputInfo('- /stats [on|off]: Enable or disable displaying of response stats');
```

Comment:
Why are we removing the option to turn stats on/off in the as a command?

Reply:
After using the tool for a while, it figured out that it clutters the command list. Most users will either ignore it or want it all time on. If you want to query the current state, you can use `/info` command to get total tokens and costs for the session.